### PR TITLE
Add redux attributes for employer register

### LINKED ISSUE #24: Implement-Redux-Data-User-Employed
## REQUIREMENTS / DESCRIPTION

```[tasklist]
### Tasks
- [ ] Configure Variable Redux
- [ ] Implement Redux for navigation data employed
- [ ] Implement Firebase for save data employed
```

### PR DESCRIPTION
**Pull request by JaimeGonz**
# Description

What did you implemented/Why did you implemented this:

- Redux was implemented to store global variables for the register employer on Firebase.

### CODE REVIEW

File: ContratApp/src/db/RegisterNewEmployer.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  let id = '';
+
+  for (let i = 0; i < longitud; i++) {
+    const indiceAleatorio = Math.floor(Math.random() * caracteres.length);
+    id += caracteres.charAt(indiceAleatorio);
+  }
+
+  return id;
+}
+
```
Comment (ecruz17) [inline ContratApp/src/db/RegisterNewEmployer.ts:26]: que no usariamos el ID generado por Firebase? xd

**Review by ecruz17 — APPROVED** [review]
Nice work!

**Comment by madera-design** [comment]
great work!

**Review by ShadowEnchanter — APPROVED** [review]
Good Job!

**Comment by ecruz17** [comment]
let's solve the conflicts to proceed with the merge

**Pull request merged**